src/DacpacTool/PackageDeployer.cs

```diff
@@ -125,9 +125,9 @@ public void RunPostDeploymentScriptFromReferences(FileInfo dacpacPackage, string
         private void RunDeploymentScriptFromReferences(FileInfo dacpacPackage, string targetDatabaseName, bool isPreDeploy)
         {
             using var model = new TSqlModel(dacpacPackage.FullName, DacSchemaModelStorageType.Memory);
-            var references = model.GetReferencedDacPackages();
+            var references = model.GetReferencedDacPackages().ToList();
```

Comment:
I don't necessarily have a problem with this, but I'm not certain why using `ToList()` and `Count` is better than using `Any()`

Reply:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1851 - I will probably revisit in #602